sips/sip-116.md

```diff
@@ -0,0 +1,137 @@
+The proposed changes involve renaming a few functions and events in two of our contracts as well as adding minor functionality to them to be able to deposit and withdraw SNX for another address. The contracts will need to be redeployed, and `SynthetixBridgeToOptimism`'s SNX balance will beed to be migrated to the new L1 bridge contract.
+
+## Motivation
+<!--This is the problem statement. This is the *why* of the SIP. It should clearly explain *why* the current state of the protocol is inadequate.  It is critical that you explain *why* the change is needed, if the SIP proposes changing how something is calculated, you must address *why* the current calculation is innaccurate or wrong. This is not the place to describe how the SIP will address the issue!-->
+As Optimism's L2 gains adoption, more and more tokens will begin to move accross the layers. One example of this is the official WETH token to be used in place of ETH in Optimism's mainnet, which will comply to the deposit/withdrawal interface standard. Unless these changes are applied, Synthetix's bridges will need to be interacted with in a different way to all the other bridges. Another reason to migrate now is that a migration in the future could involve the migration of a much larger SNX value.
+
+## Specification
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+Migrate our L1 and L2 bridges, `SynthetixBridgeToOptimism` and `SynthetixBridgeToBase` to conform to Optimism's standard deposit/withdrawal interface.
+
+### Rationale
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+First, changes should be made to `SynthetixBridgeToOptimism` and `SynthetixBridgeToBase`. Following that, the corresponding test cases should be updated, and tests should be written to cover new functionality. Finally, the two contracts should be deployed on each layer, and linked to the system. The bridge in L1 is particularly sensitive because it locks a large number of funds.
+
+### Technical Specification
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
```

Comment:
Can you link to eth-optimism's interface?

Reply:
Done